### ISSUE TITLE
This is not a very useful guide for customising the Amplify Auth screens since none of these appear to work on the default react application

### ISSUE THREAD

**Issue by duncangroenewald** [aws-amplify/docs#2473]
**Page**: [`/lib/auth/customui/q/platform/js`](https://docs.amplify.aws/lib/auth/customui/q/platform/js)

**Feedback**:

This guide for customising the Auth screens UI is pretty useless since none of these suggestions have sufficient context and don't appear to work when used with a standard AWS Amplify react serveless application.

By default the login screens have not styling - which is not what is shown in any of the tutorial screens.  It seems it is necessary to import the styling with the following line in the default created application:

import '@aws-amplify/ui/dist/style.css';

And then none of the customisations appear to work - but there is no explanation of what files these theme customisations should be made in and there appear to be more than one way to change the styling of the auth screens - although none appear to work.

AWS Amplify looks promising but having guides for customisation that don't actually work is pretty frustrating and leaves one thinking the whole thing is still very much in beta.

**Comment by mauerbac** [comment]
Hey @duncangroenewald . Sorry to hear the guide isn't helpful. Just so i'm clear - is hosted UI what you are referring to by "having not styling" 
![image](https://user-images.githubusercontent.com/1507886/94724252-fee59a00-0327-11eb-8ed4-16d32af9625b.png)

You have linked the docs that explains how you can build your own UI. Can you expand on what should be made more clear around themes/customizations ?

**Comment by duncangroenewald** [comment]
Sorry that was meant to read “having no styling”.  By default when I created a new app and run it in development mode as per the Amplify tutorials the login screen shown below has no styling at all.  So it does not look like the screen below.  Once I added the import line as per my feedback then the styling shown in the image below is used and the screen looks just like the one below.

Without that line the page shows up with default text, text entry fields and buttons all squashed up in the top left corner.  Just as if it was plain html with no formatting at all.

So that is the first problem.

And then following the instructions to change the style of the login screens by using another custom theme also appears to have no effect - regardless of which of the options I tried.  Having said that, it is unclear from the instructions exactly where all the changes that are shown are supposed to be made so I may have done then in the wrong places - i.e. which files needs to be edited and where exactly in the file are the new lines supposed to be added.  

I think perhaps the guide as written assumes the reader has some reasonably detailed knowledge of javascript or react or something.  

Also I am not sure what you mean by ‘hosted UI’ ?

Regards
